src/main/java/uk/gov/indexer/IndexerTask.java

```diff
@@ -28,9 +28,7 @@ public IndexerTask(Optional<CloudwatchRecordsProcessedUpdater> cloudwatchUpdater
     @Override
     public void run() {
         try {
-            LOGGER.info("Starting update for: " + register);
             update();
-            LOGGER.info("Finished for register: " + register);
         } catch (Throwable e) {
```

Comment:
I have removed the Log verbosity related commit from this PR. will raise new PR for that.